### DISCUSSION TITLE
New triager: William Lachance

### DISCUSSION THREAD

**Discussion by AA-Turner** [sphinx-doc/sphinx#12328] [Project Announcements]
Please welcome @wlach as a new triager for Sphinx -- Will has recently been doing stellar work on search & JS.

A

**Comment by chrisjsewell** [comment]
Hey @wlach 👋 

@AA-Turner, @sphinx-doc/developers @sphinx-doc/triagers (and really anyone else), as we now have a good few people added to the "team", I wonder if you would be interested in having an online meet sometime?

Nothing formal, I just find it is nice sometimes to put a face to a GitHub tag 😄 

(Me and @picnixz actually met a few months back at EPFL (Switzerland) and had a nice chat in person)

**Reply by jdillard** [reply]
I'd be up for a online meet! I'm UTC-7 and flexible on time.

**Reply by wlach** [reply]
Up for it too! UTC-5 here. I'll also be at PyCon US this year if anyone is going.

**Reply by chrisjsewell** [reply]
Hey @jdillard @wlach (and anyone else), how about next Wednesday 1800 UTC then?
(or propose some other day/times you'd be available)

**Reply by wlach** [reply]
Next week is a little hectic as I'm trying to wrap up some work stuff before PyCon. Might make sense to plan something a few weeks out and set up a doodle/google poll to get timing? In general Fridays are better for me.

**Reply by shimizukawa** [reply]
I am also interested, but since I am in a different time zone than you all (UTC+9), I will try to attend the event as long as the time is set and I can attend.

**Reply by chrisjsewell** [reply]
Ok guys let's give this a go! (sorry for the delay): here is the doodle https://doodle.com/meeting/participate/id/ax74pkle

**Reply by chrisjsewell** [reply]
Made a new discussion: https://github.com/orgs/sphinx-doc/discussions/12443
